ghdocs/TUTORIAL.md

```diff
@@ -154,46 +155,68 @@ Go ahead and refresh—the extra white space along the top and sides should go a
 
 ## Step 4 – Using Fabric components
 
-Fabric includes a responsive grid system, similar to those seen in other UI toolkits, that we will use for our TodoBar section. Let's add the following HTML inside  `<div class="TodoBar"></div>`:
+Fabric includes a responsive grid system, similar to those seen in other UI toolkits, that we will be using for our TodoBar section. Let's add the following HTML inside  `<div class="TodoBar"></div>`:
 
 ```html
 <div class="ms-Grid">
   <div class="ms-Grid-row">
-    <div class="ms-Grid-col ms-u-sm6 ms-u-md8 ms-u-lg10"></div>
-    <div class="ms-Grid-col ms-u-sm6 ms-u-md4 ms-u-lg2"></div>
+    <div class="ms-Grid-col ms-u-sm6 ms-u-md8 ms-u-lg10"></div> <!-- First Column -->
+    <div class="ms-Grid-col ms-u-sm6 ms-u-md4 ms-u-lg2"></div> <!-- Second Column -->
   </div>
 </div>
 ```
 
-Now for the fun part: let's add some component HTML into our ToDo app. Navigate to where you extracted your .zip of Fabric, then open the following component files in your code editor:
+Now for the fun part: let's add some component HTML into our ToDo app. Navigate to where you extracted Fabric, then open the following component files in your code editor:
 
 ```
 dist/components/Button/Button.html
 dist/components/TextField/TextField.html
 ```
 
-First, copy all the contents from TextField.html and paste it into the first `.ms-Grid-col ` `<div>`, then remove the `<label>` element. The HTML should look now like this:
+First, copy all the contents from TextField.html and paste it inside of the first column `<div class="ms-Grid-col"></div> <!-- First Column -->`, then remove the `<label class="ms-Label">Name</label>` element and `<span class="ms-TextField-description">This should be your first and last name.</span>`.  At this point your Second Column HTML should look like this:
```

Comment:
I would pick to capitalize First Column if you plan to capitalize Second Column for consistency's sake.